### PR TITLE
Change the support email to a link to the support page.

### PR DESCRIPTION
**Pull request by mwasn**

### CODE REVIEW

File: getting-started/privacy.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Remember that your use of Mesosphere’s Services is at all times subject to the
...
+As noted in the <a href="/getting-started/terms">Terms of Use</a>, we do not knowingly collect or solicit personal information from anyone under the age of 13. If you are under 13, please do not attempt to register for the Services or send any personal information about yourself to us. If we learn that we have collected personal information from a child under age 13, we will delete that information as quickly as possible. If you believe that a child under 13 may have provided us personal information, please contact us at one of our <a href="https://docs.mesosphere.com/support/">support channels</a>.
```
Comment (ssk2) [inline getting-started/privacy.md]: "please contact us at one" should be "please contact us through one"

File: tutorials/run-hadoop-on-mesos.html [hunk 2]
```diff
@@ -23,7 +23,7 @@
...
       <p>Hadoop can take advantage of the elasticity and fault tolerance Mesos provides. Running Hadoop on top of Mesos lets you share your cluster resources with other frameworks to yield higher utilization.</p>
       <p>Mesos is made up of a set of master nodes and a set of slaves. For this tutorial, we're running the Hadoop <a href="http://wiki.apache.org/hadoop/JobTracker">JobTracker</a> on one of the master nodes and the TaskTrackers will be launched on slave nodes. We will demonstrate how to configure Hadoop to leverage Mesos directly.</p><br />
-      <p>Some systems are setup differently than others and certain tools may not be available. We're doing our best to make this an easy and fun experience. If you have trouble getting going, please feel free to email us at <a href="mailto:support@mesosphere.io?Subject=Hadoop%20tutorial">support@mesosphere.io.</a></p>
+      <p>Some systems are setup differently than others and certain tools may not be available. We're doing our best to make this an easy and fun experience. If you have trouble getting going, please feel free to to contact us at our <a href="https://docs.mesosphere.com/support/">support channels.</a></p>
```
Comment (ssk2) [inline tutorials/run-hadoop-on-mesos.html]: "at" -> "through"

**Comment by ssk2** [comment]
Thanks @mwasn! You might need to rebase on top of master.

**Comment by mwasn** [comment]
@ssk2 Conflicts resolved. Ready for review and merge.

**Pull request merged**